languages/en.php

```diff
@@ -1294,22 +1294,29 @@
 	'generic_comment:on' => '%s on %s',
 	'generic_comments:latest:posted' => 'posted a',
 
-	'generic_comment:email:subject' => 'You have a new comment!',
-	'generic_comment:email:body' => "You have a new comment on your item \"%s\" from %s. It reads:
-
+	'generic_comment:notification:owner:subject' => 'You have a new comment!',
+	'generic_comment:notification:owner:summary' => 'You have a new comment!',
+	'generic_comment:notification:owner:body' => "You have a new comment on your item \"%s\" from %s. It reads:
```

Comment:
In docs/guides/upgrading.rst add instructions for plugin devs to migrate their language keys. We probably want to mention this in docs/admin/upgrading.rst, too.